test/answer.e2e-spec.ts

```diff
@@ -148,61 +149,83 @@ describe('Answers Module', () => {
+          .set('Authorization', `Bearer ${auxAccessToken}`)
+          .send({ content });
+        expect(respond.body.message).toBe('Answer created successfully.');
+        expect(respond.body.code).toBe(200);
+        expect(respond.status).toBe(201);
+        expect(typeof respond.body.data.id).toBe('number');
+        answerId.push(respond.body.data.id);
+        AnswerQuestionMap[respond.body.data.id] = questionId;
+      }
+      const answerContents = [
+        '你说得对，但是原神是一款由米哈游自主研发的开放世界游戏，后面忘了',
+        '难道你真的是天才？',
+        '你不要胡说，1+1明明等于3',
+        'Answer content with emoji: 😂😂',
+        '烫烫烫'.repeat(1000),
+      ];
+      await Promise.all(
+        questionId.map((id, index) => createAnswer(id, answerContents[index])),
+      );
+    }, 60000);
+    it('should return QuestionAlreadyAnsweredError when user answer the same question', async () => {
+      const TestQuestionId = questionId[0];
+      const content = 'content';
+      await request(app.getHttpServer())
+        .post(`/questions/${TestQuestionId}/answers`)
+        .set('Authorization', `Bearer ${auxAccessToken}`)
+        .send({ content });
       const respond = await request(app.getHttpServer())
-        .post(`/questions/${testQuestionId}/answers`)
+        .post(`/questions/${TestQuestionId}/answers`)
         .set('Authorization', `Bearer ${auxAccessToken}`)
-        .send({
-          content: content,
-        });
-      expect(respond.body.message).toBe('Answer created successfully.');
-      expect(respond.body.code).toBe(200);
-      expect(respond.status).toBe(201);
-      expect(typeof respond.body.data.id).toBe('number');
-      answerId.push(respond.body.data.id);
-    }
-    await createAnswer(
-      '你说得对，但是原神是一款由米哈游自主研发的开放世界游戏，后面忘了',
-    ); // this should be firstly executed and will be checked further
-    await Promise.all([
-      createAnswer('难道你真的是天才？'),
-      createAnswer('你不要胡说，1+1明明等于3'),
-      createAnswer('Answer content with emoji: 😂😂'),
-      createAnswer('烫烫烫'.repeat(1000)),
-    ]);
-  }, 6000);
-  it('should return updated statistic info when getting user', async () => {
-    const respond = await request(app.getHttpServer()).get(
-      `/users/${auxUserId}`,
-    );
-    expect(respond.body.data.user.answer_count).toBe(5);
-  });
-  it('should return updated statistic info when getting user', async () => {
-    const respond = await request(app.getHttpServer())
-      .get(`/users/${auxUserId}`)
-      .set('authorization', 'Bearer ' + TestToken);
-    expect(respond.body.data.user.answer_count).toBe(5);
+        .send({ content });
+      expect(respond.body.message).toMatch(/QuestionAlreadyAnsweredError: /);
+      expect(respond.body.code).toBe(400);
+    });
+    it('should return updated statistic info when getting user', async () => {
+      const respond = await request(app.getHttpServer()).get(
+        `/users/${auxUserId}`,
+      );
+      expect(respond.body.data.user.answer_count).toBeDefined();
```

Comment:
You can check it actually. Since you could get an userDto `auxUserDto` when registering the user. See group-e2e `createAuxiliaryUser`.
e.g. expect answer_count == auxUserDto.answer_count + 5